docs/LanguageComparison.md

```diff
@@ -5,94 +5,98 @@
 | **API Endpoints** | | | |
-| Implemented /api/movies?q={}&genre={}&year={}&rating={}&actorId={}&pageNumber={}$pageSize={} | Yes | Yes |  |
-| Implemented /api/movies/{movieId} | Yes | Yes |  |
-| Implemented /api/actors?q={}&pageNumber={}$pageSize={} | Yes | Yes |  |
-| Implemented /api/actors/{actorId} | Yes | Yes |  |
-| Implemented /api/featured/movie | Yes | Yes |  |
-| Implemented /api/genres | Yes | Yes |  |
+| Implemented /api/movies?q={}&genre={}&year={}&rating={}&actorId={}&pageNumber={}$pageSize={} | Yes | Yes | Yes |
+| Implemented /api/movies/{movieId} | Yes | Yes | Yes |
+| Implemented /api/actors?q={}&pageNumber={}$pageSize={} | Yes | Yes | Yes |
+| Implemented /api/actors/{actorId} | Yes | Yes | Yes |
+| Implemented /api/featured/movie | Yes | Yes | Yes |
+| Implemented /api/genres | Yes | Yes | Yes |
 | |
 | **Middleware** | | | |
-| /robots**.txt is handled | Yes | Yes |  |
-| /version returns app version | Yes | Yes |  |
+| /robots**.txt is handled | Yes | Yes | Yes |
+| /version returns app version | Yes | Yes | Yes |
 | |
 | **Healthz** | | | |
-| Endpoint /healthz returns pass/warn/fail | Yes | Yes |  |
-| Endpoint /healthz/ietf returns expected ietf response | Yes | Yes | |
+| Endpoint /healthz returns pass/warn/fail | Yes | Yes | Yes |
+| Endpoint /healthz/ietf returns expected ietf response | Yes | Yes | Yes |
 | Idiomatic healthz endpoint implemented | Yes | na | na |
 | |
 | **CI-CD/Build/Deployment** | | | |
-| Includes Dockerfile at root | Yes | Yes |  |
-| Includes Dockerfile-Dev that runs locally with MSI | Yes | Yes |  |
-| CI-CD is set up in repo | Yes | Yes |  |
-| CI-CD includes e2e testing | **No** | **No** |  |
-| Can be deployed with App Services | Yes | Yes |  |
-| Can be deployed with AKS (using Pod Identity) | Yes | **No** |  |
+| Includes Dockerfile at root | Yes | Yes | Yes |
+| Includes Dockerfile-Dev that runs locally with MSI | Yes | Yes | **No** |
+| CI-CD is set up in repo | Yes | Yes | Yes |
+| CI-CD includes e2e testing | **No** | **No** | **No** |
+| Can be deployed with App Services | Yes | Yes | Yes |
+| Can be deployed with AKS (using Pod Identity) | Yes | Yes | **No**  |
 | |
 | **Linting** | | | |
-| Linter being used | fxcop | eslint |  |
-| **No** lint errors | Yes | Yes |  |
-| Linter running in Dockerfile, fails to build if errors | Yes | Yes |  |
+| Linter being used | fxcop | eslint | checkstyle, pmd, cpd and findbugs |
+| **No** lint errors | Yes | Yes | Yes  |
+| Linter running in Dockerfile, fails to build if errors | Yes | Yes | Yes |
 | |
 | **Documentation** | | | |
-| Developer docs | **No** | **No** |  |
-| Readme specific to language | Yes | Yes |  |
-| Swagger Docs | Yes | Yes |  |
-| Publish and use shared Swagger | **No** | **No** |  |
-| |
+| Developer docs | Yes | Yes | **No** |
+| Readme specific to language | Yes | Yes | Yes |
+| Swagger Docs | Yes | Yes | Yes |
+| Publish and use shared Swagger | Yes | Yes | Yes |
+|
 | ***Managed Identity + Key Vault (MIKV)*** |
```

Comment:
The MIKV part doesn't render properly,  I think line 99 is missing an additional " |  ".